docs/rules/no-duplicate-imports.md

```diff
@@ -14,57 +16,47 @@ import { find } from 'module';
 
 This inspection reports any duplicated module in an import statement.
```

Comment:
I think he means:

Move "This rules requires that all imports from a single module exists in a single import statement." sentence from top of document down in to the **Rule Details** section (so it becomes first sentence in the Rule Details section).